libs/features/personalization/personalization.js

```diff
@@ -540,6 +540,25 @@ export function parseManifestVariants(data, manifestPath, targetId) {
   return null;
 }
 
+export async function createMartechMetadata(placeholders, config, column) {
+  if (config.locale.ietf === 'en-US') return;
+
+  await import('../../martech/attributes.js').then(({ processTrackingLabels }) => {
```

Comment:
I'd be a bit careful with those imports as they block the pipeline and we can't load things in parallel.

Reply:
Here's a bit of a case-study I did on project unity on this: https://github.com/adobecom/unity/pull/102